src/_node/apis.ts

```diff
@@ -345,7 +336,7 @@ export const copyNode = (
     newNode.parentUid = targetUid;
 
     if (treeType === "html") {
-      setHtmlNodeInAppAttribName(newNode, newUid);
+      setHtmlNodeUidAttribNameInApp(newNode, newUid);
```

Comment:
can we rename it to a much easy node so that we can remember it when we reference it. Like `stageViewNodeId` instead of `htmlNodeUidAttribNameInApp`

and instead of having the value of the following constant `data-rnbwdev-rnbw-node-uid` can we have it like: `data-rnbw-nide-uid`

Reply:
sure, I'll improve it.